eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -140,6 +140,10 @@ jobs:
           $(Build.SourcesDirectory)/eng/versioning/pom_file_version_scanner.ps1
         displayName: "Verify versions in POM files"
 
+      - template: ../../../common/pipelines/templates/steps/verify-links.yml
```

Comment:
This should be lower in the list of verifications. We should verify POM files then README then CHANGELOGs and then do the link verification. It doesn't make sense that would we verify links before verifying the content of READMEs and CHANGELOGs.